stories/create-stories-from-doc.js

```diff
@@ -0,0 +1,149 @@
+    parameters: {\n\
+      docs: {\n\
+        page: CustomMDXDocumentation,\n\
+      }\n\
+    },\n\
+  }\n\
+  \n\
+  export const ' + component + '_' + index + ' = () => `' + content + '`'
+}
+
+const convertToKebabCase = string => {
+  return string.replace(/([a-z])([A-Z])/g, '$1-$2')
+    .replace(/\s+/g, '-')
+    .toLowerCase()
+}
+
+// TODO: build this list automatically from docs
+const files = [
+  ['Accordion', '../_site/docs/5.2/components/accordion/index.html'],
+  ['Alerts', '../_site/docs/5.2/components/alerts/index.html'],
+  ['BackToTop', '../_site/docs/5.2/components/back-to-top/index.html'],
+  ['Badge', '../_site/docs/5.2/components/badge/index.html'],
+  ['Breadcrumb', '../_site/docs/5.2/components/breadcrumb/index.html'],
+  ['ButtonGroup', '../_site/docs/5.2/components/button-group/index.html'],
+  ['Buttons', '../_site/docs/5.2/components/buttons/index.html'],
+  ['Card', '../_site/docs/5.2/components/card/index.html'],
+  ['Carousel', '../_site/docs/5.2/components/carousel/index.html'],
+  ['CloseButton', '../_site/docs/5.2/components/close-button/index.html'],
+  ['Collapse', '../_site/docs/5.2/components/collapse/index.html'],
+  ['Dropdowns', '../_site/docs/5.2/components/dropdowns/index.html'],
+  ['Footer', '../_site/docs/5.2/components/footer/index.html'],
+  ['ListGroup', '../_site/docs/5.2/components/list-group/index.html'],
+  ['Modal', '../_site/docs/5.2/components/modal/index.html'],
+  ['Navbar', '../_site/docs/5.2/components/navbar/index.html'],
+  // ['Navs', '../_site/docs/5.2/components/navs/index.html'],
+  ['NavsTabs', '../_site/docs/5.2/components/navs-tabs/index.html'],
+  ['OffCanvas', '../_site/docs/5.2/components/offcanvas/index.html'],
+  ['OrangeNavbar', '../_site/docs/5.2/components/orange-navbar/index.html'],
+  ['Pagination', '../_site/docs/5.2/components/pagination/index.html'],
+  ['Popovers', '../_site/docs/5.2/components/popovers/index.html'],
+  ['Progress', '../_site/docs/5.2/components/progress/index.html'],
+  ['Scrollspy', '../_site/docs/5.2/components/scrollspy/index.html'],
+  ['Spinners', '../_site/docs/5.2/components/spinners/index.html'],
+  ['SteppedProcess', '../_site/docs/5.2/components/stepped-process/index.html'],
+  ['Sticker', '../_site/docs/5.2/components/sticker/index.html'],
+  ['Tags', '../_site/docs/5.2/components/tags/index.html'],
+  ['Toasts', '../_site/docs/5.2/components/toasts/index.html'],
+  ['TitleBars', '../_site/docs/5.2/components/title-bars/index.html'],
+  ['Tooltips',
+    '../_site/docs/5.2/components/tooltips/index.html',
+    'var tooltipTriggerList = [].slice.call(document.querySelectorAll(\'[data-bs-toggle="tooltip"]\'));\
+    var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {\
+      return new boosted.Tooltip(tooltipTriggerEl);\
+    })']
+]
+
+const outputDirectory = __dirname + '/auto'
+createDirectoryIfNeeded(outputDirectory);
+
+(async () => {
+  const browser = await puppeteer.launch()
```

Comment:
It doesn't work on my side. I have the same error as you when we talked about it this morning:

```
"file:/Users/ju/Orange-Boosted-Bootstrap/_site/storybook/index.html":1.38-1.60: info warning: Self-closing tag syntax in text/html documents is widely discouraged; it's unnecessary and interacts badly with other HTML features (e.g., unquoted attribute values). If you're using a tool that injects self-closing tag syntax into all void elements, without any option to prevent it from doing so, then consider switching to a different tool.
"file:/Users/ju/Orange-Boosted-Bootstrap/_site/storybook/index.html":1.87-1.154: info warning: Self-closing tag syntax in text/html documents is widely discouraged; it's unnecessary and interacts badly with other HTML features (e.g., unquoted attribute values). If you're using a tool that injects self-closing tag syntax into all void elements, without any option to prevent it from doing so, then consider switching to a different tool.
ERROR: "docs-lint" exited with 1.
```